man/magellan.1.sc

```diff
@@ -79,11 +79,102 @@ The *magellan* command accepts
 
 ## GETTING STARTED
 
-Nothing here yet.
+The *magellan* CLI is a frontend tool for collecting inventory data from board
+management controllers (BMCs) through a running Redfish service. The tool may be 
+used in the following workflows:
+
+
+1. Simple Workflow: 	scan -> collect -> send
+2. Complex Workflow: 	scan -> list -> secrets -> collect -> send
+3. Collect Workflow:	scan -> collect -> send -> collect -> send -> *
+4. Crawl Workflow:		crawl -> send -> crawl -> send -> *
+
+### Simple Workflow
```

Comment:
```
Error at 92:4: Only headings up to two levels deep are permitted
```

Could we move:
- `## GETTING STARTED` -> `# GETTING STARTED`
- `### * Workflow` -> `## * Workflow`

?